project/Dependencies.scala

```diff
@@ -131,7 +131,7 @@ object Dependencies {
   val sealerate: ModuleID =       "ca.mrvisser"         %% "sealerate"            % "0.0.6"
   val googleCloudNio: ModuleID =  "com.google.cloud"    % "google-cloud-nio"      % "0.126.15" % Test // brought in for FakeStorageInterpreter
 
-  val circeYaml =         "io.circe"          %% "circe-yaml"           % "0.14.2"
+  val circeYaml =         "io.circe"          %% "circe-yaml"           % "0.15.0-RC1"
```

Comment:
@Qi77Qi Is this fine to use an RC?
It fixes a vulnerability from a dependency of snakeyaml
Here's the RC being used:
https://mvnrepository.com/artifact/io.circe/circe-yaml_2.13/0.15.0-RC1

Here's the version that imports vulnerable versions.
https://mvnrepository.com/artifact/io.circe/circe-yaml_2.13/0.14.2

Reply:
Probably fine if automation tests are all passing